src/content/structured/accessibility/needs/neurodiversity.mdx

```diff
@@ -156,18 +161,55 @@ Some people find it easier to look at images. Use graphs, icons and colour along
 
 All images and icons must use [accessible colour](/styles/colour) and have `alt` (alternate) text and captions.
 
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={neurodiversityAccessibilityFig14}
+    imageAlt="A graphic of a survey regarding eye colour. Results are conveyed by short lines of text and by a pie chart that displays the results in a more visible solution."
+    state="good"
+    caption="Using both images and text can make it easier to understand data."
+  />
+  <DoDontCaution
+    imageSrc={neurodiversityAccessibilityFig15}
+    imageAlt="A graphic of a survey regarding eye colour. Results are written in long sentences that do not show the information as clearly as possible."
+    state="bad"
+    caption="Using text only to present data can be complex and overwhelming."
+  />
+</DoubleDoDontCaution>
+
+<IcAlert
+  heading=""
+  message="The pie chart in the diagram above is not a component, but shows how images should be imported. 
+  Please ensure that any text in the diagram is accessible with a screen reader."
+  variant="info"
+/>
+
 ### Don't expect the user to remember
 
 If you use multiple input steps, make sure you remind users what they've entered before. Don't expect people to switch (or remember) between screens to see what they typed in previously.
 
-### Be flexible on spelling
+### Use simple words
 
-Where possible, your app should cope with different date formats.
-
-Use auto-complete suggestion where relevant so that people don't have to spell out the full text.
+Where possible, use [simple words](/styles/content-style/voice-and-tone) no harder than a reading age of 9.
```

Comment:
Suggested change - "Where possible, use simple words and aim for a maximum reading age of 9."